src/components/Tool/NavToolList/Item/Item.test.tsx

```diff
@@ -12,24 +12,45 @@ const item = {
   id: 'testItem',
 };
 
+const TestComponent = ({ value = 0 }: { value?: number }) => (
```

Comment:
How about giving this prop the same name as the `Item` prop it represents?

```suggestion
const TestComponent = ({ totalCount = 0 }: { totalCount?: number }) => (
```